packages/user/src/supertokens/recipes/config/third-party-email-password/sendPasswordResetEmail.ts

```diff
@@ -15,10 +15,19 @@ const sendPasswordResetEmail = (
   const websiteDomain = fastify.config.appOrigin[0] as string;
 
   return async (input) => {
-    const request: FastifyRequest = input.userContext._default.request.request;
+    const request: FastifyRequest<{
+      Querystring: { appId: string | undefined };
+    }> = input.userContext._default.request.request;
+
+    const appId = Number(request.query.appId);
+
+    const app = fastify.config.apps?.find((app) => app.id === appId);
```

Comment:
can we run this code only if appId is defined.

Reply:
:+1: